pkgs/top-level/aliases.nix

```diff
@@ -1649,7 +1651,7 @@ mapAliases ({
   openbsdCross = openbsd; # Added 2024-06-18
 
   # LLVM packages for (integration) testing that should not be used inside Nixpkgs:
-  llvmPackages_latest = llvmPackages_18;
+  llvmPackages_latest = llvmPackages_19;
```

Comment:
Would be a policy change to set this to an rc version.  Usually we update this immediately when the new version comes out (on master), since getting the default updated can take a while (even if nowadays it should only be a couple of weeks).

Reply:
I thought it would be fine since `llvmPackages_latest` isn't really used for anything.
```
pkgs/top-level/aliases.nix:  libclc = llvmPackages_latest.libclc; # Added 2023-10-28
pkgs/top-level/aliases.nix:  llvmPackages_latest = llvmPackages_19;
pkgs/test/default.nix:        (filter (n: n != "llvmPackages_latest"))
```

Even libclc doesn't really do much.
```
pkgs/top-level/aliases.nix:  libclc = llvmPackages_latest.libclc; # Added 2023-10-28
pkgs/development/compilers/llvm/common/libclc/libclc-gnu-install-dirs.patch:--- a/libclc.pc.in
pkgs/development/compilers/llvm/common/libclc/libclc-gnu-install-dirs.patch:+++ b/libclc.pc.in
pkgs/development/compilers/llvm/common/libclc/libclc-gnu-install-dirs.patch: Name: libclc
pkgs/development/compilers/llvm/common/default.nix:      libclc = callPackage ./libclc.nix { };
pkgs/development/compilers/llvm/common/libclc.nix:  pname = "libclc";
pkgs/development/compilers/llvm/common/libclc.nix:    ./libclc/libclc-gnu-install-dirs.patch
pkgs/development/compilers/llvm/common/libclc.nix:  ] ++ lib.optional (lib.versionAtLeast release_version "19") ./libclc/find-program.patch;
pkgs/development/compilers/llvm/common/libclc.nix:        --replace 'COMMAND prepare_builtins' 'COMMAND ${buildLlvmTools.libclc.dev}/bin/prepare_builtins'
pkgs/development/compilers/llvm/common/libclc.nix:    homepage = "http://libclc.llvm.org/";
pkgs/development/libraries/mesa/default.nix:    llvmPackages.libclc
pkgs/development/rocm-modules/6/llvm/default.nix:  libclc = callPackage ./stage-3/libclc.nix { inherit rocmUpdateScript llvm clang; stdenv = rocmClangStdenv; };
pkgs/development/rocm-modules/6/llvm/stage-3/libclc.nix:  targetName = "libclc";
pkgs/development/rocm-modules/5/llvm/default.nix:  libclc = callPackage ./stage-3/libclc.nix { inherit rocmUpdateScript llvm clang; stdenv = rocmClangStdenv; };
pkgs/development/rocm-modules/5/llvm/stage-3/libclc.nix:  targetName = "libclc";
```

Reply:
The purpose for it is to be used out of tree — I think users probably expect "llvmPackages_latest" to keep referring to the latest released version of LLVM, like it always has done, rather than a release candidate.

Reply:
Gotcha, thank you for the explanation. I will remove that change.

Reply:
It has been removed.